swagger.json

```diff
@@ -664,6 +664,60 @@
         }
       }
     },
+    "/api/volunteers/{id}": {
```

Comment:
good job documenting `GET/ volunteers/:id` but also we need to document `GET/ volunteers` as well